tox.ini

```diff
@@ -105,9 +105,11 @@ deps =
     nidcpower_system_tests: enum34;python_version<"3.4"
     nidcpower_system_tests: pytest
     nidcpower_system_tests: nidcpower
+    nidcpower_system_tests: pytest-json
     nidmm_system_tests: enum34;python_version<"3.4"
     nidmm_system_tests: pytest
     nidmm_system_tests: nidmm
+    nidmm_system_tests: pytest-json
     niswitch_system_tests: enum34;python_version<"3.4"
     niswitch_system_tests: pytest
     niswitch_system_tests: niswitch
```

Comment:
Do you need to add pytest-json to switch?

Reply:
Currently internal testsuite for niswitch won't report results in elasticsearch. We planned to have a link to internal testsuite report aswell for DMM and DCPower since it upload results to elasticsearch servers.

But I think it is good to have a centralized location for all driver testsuite results. I have updated tox to have pytest-json dependency for switch aswell. It should update results to same elasticserver as nidmm and nidcpower.